tests/functional/local/apigw/test_service.py

```diff
@@ -450,6 +502,33 @@ def test_post_binary_with_incorrect_content_type(self):
         self.assertEquals(response.headers.get('Content-Type'), "application/json")
 
 
+class TestService_CaseInsensiveDict(TestCase):
```

Comment:
Unit tests should go in the `tests/unit/local/apigw/test_service.py` not in the `tests/functional`. Can you move this to the correct folder/file?